hudi-common/src/main/java/org/apache/hudi/common/table/TableSchemaResolver.java

```diff
@@ -71,7 +72,7 @@
  * Helper class to read schema from data files and log files and to convert it between different formats.
  */
 @ThreadSafe
-public class TableSchemaResolver {
+public class TableSchemaResolver implements Serializable {
```

Comment:
Why does `TableSchemaResolver` have to be Serializable?

Reply:
Good catch! Addressed the issue.